tests/datasource/fluent/test_config.py

```diff
@@ -488,6 +489,28 @@ def test_yaml_file_config_round_trip(
     assert from_yaml_gx_config == re_loaded
 
 
+def test_assets_key_presence(
+    inject_engine_lookup_double, from_yaml_gx_config: GxConfig
+):
+    ds_wo_assets = None
+    ds_with_assets = None
+    for ds in from_yaml_gx_config.datasources.values():
+        if not ds.assets:
```

Comment:
@Kilo59 Non-blocking: stylistically, when the `if` clause contains the `else`, it might be less verbose to put the negative branch into the `else` part.  Thanks.

Reply:
I agree.

Reply:
🙇